src/pages/gift/accounts/AccountOverview.js

```diff
@@ -23,12 +26,24 @@ const AccountField = ({ title, value }) => {
   );
 };
 const WalletInfoField = ({ title, subtitle, ...props }) => {
+  const [logo, setLogo] = useState('');
+
+  useEffect(() => {
```

Comment:
You might not need to keep the logo in state and useEffect, since it is loaded based on title property and won't change during component lifecycle. can only load it using some if (title==='...') conditions

Reply:
True! That way it is also more consistent with the card buttons